### PR TITLE
Show error list in blocks editor as experiment

### PR DESCRIPTION
**Pull request by hristoramirez**
Starting the blocks error list experiment. Still working on getting all errors to appear. 
There are still a couple of issues with the layout
See: microsoft/pxt-arcade#2003
blocksErrorlistExperiment

### CODE REVIEW

**Comment by abchatra** [comment]
We should merge this soon.

**Comment by abchatra** [comment]
@darzu may be next week.

File: theme/blockly-core.less [hunk 1]
```diff
@@ -18,6 +18,18 @@ svg.blocklySvg {
     background: @blocklySvgColor;
 }
 
+.blocksAndErrorList {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+}
+.blocksEditorOuter {
+    position: relative !important;
```
Comment (darzu) [inline theme/blockly-core.less:27]: slightly concerning

**Comment by darzu** [comment]
I'd prefer to finish this post Arcade release. This will require some regression testing of viewports on different devices since we're changing the html and css around the main blocks editor.

**Comment by abchatra** [comment]
Yes, makes sense to me. Lets wait for arcade release.

**Comment by abchatra** [comment]
@darzu time to push this through.

**Comment by darzu** [comment]
Tracking: https://github.com/microsoft/pxt-arcade/issues/2310

**Comment by abchatra** [comment]
@darzu what is the status of this?

**Comment by darzu** [comment]
- [ ] Crash on open error list:

File: theme/blockly-core.less (hunk 1 above)
Comment (darzu) [reply theme/blockly-core.less:27]: removed and everything seems okay.

**Comment by darzu** [comment]
Did a bit of testing and everything seems okay but I still think there is some risk of regression.

**Review by darzu — APPROVED** [review]

**Review by eanders-MS — COMMENTED** [review]
lgtm

**Review by eanders-MS — APPROVED** [review]
lgtm

**Pull request merged**